tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -170,7 +170,7 @@ public boolean booleanValue() {
   }
 
   /**
-   * Returns the value in a scalar {@link DataType#STRING} tensor.
+   * Returns the value in a scalar {@link DataType#STRING} Tensor.
```

Comment:
I feel that "tensor" and not "Tensor" is appropriate here and in many of the other places.
The idea being that "Tensor" is used when referring to the Java class or objects that are instances of it. While "tensor" is used for the conceptual N-dimensional array.

One could make the case that the Tensor object itself is not a "scalar" or a "vector", hence "scalar Tensor" isn't meaningful while "scalar tensor" is.

That said, I don't feel too strongly about this and I probably wasn't very consistent in the distinction between 'tensor' and 'Tensor' in comments. 

If you feel particularly strongly, I'd appreciate your thoughts on why.